R/reportClimate.R

```diff
@@ -0,0 +1,37 @@
+#' Read in GDX and extract climate assessment variables
+#'
+#' Read climate assessment variables from GDX file
+#'
+#' @param gdx a GDX as created by readGDX, or the file name of a gdx
+#'
+#' @author Tonn Rüter
+#' @examples
+#' \dontrun{
+#' reportClimate(gdx)
+#' }
+#'
+#' @export
+#' @importFrom gdx readGDX
+#' @importFrom magclass setNames mbind
+reportClimate <- function(gdx) {
+  tryCatch(
```

Comment:
tryCatch can most likely be avoided here. `readGDX` has a `react` parameter  that can be set to `silent` to return NULL

Reply:
Done